src/fork.rs

```diff
@@ -61,7 +76,7 @@ impl ForkStorage {
                 raw_storage: InMemoryStorage::with_system_contracts_and_chain_id(
                     chain_id,
                     hash_bytecode,
-                    dev_use_local_contracts,
```

Comment:
We need to keep this parameter to maintain functionality of using local bootloader/system contracts